packages/yices2_bindings/yices2_bindings.0.1/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "Ocaml bindings for yices2"
+description: "Ocaml bindings for yices2"
+maintainer: "Stephane Graham-Lengrand <stephane.graham-lengrand@csl.sri.com>"
+authors: "Stephane Graham-Lengrand <stephane.graham-lengrand@csl.sri.com>"
+license: "GPLv3"
+homepage: "https://github.com/SRI-CSL/yices2_ocaml_bindings"
+bug-reports: "https://github.com/SRI-CSL/yices2_ocaml_bindings/issues"
+depends: [
+  "ocaml" {>= "4.08" & < "4.11"}
```

Comment:
are you certain this package is not compilable with OCaml 4.11? If you're not, could you remove the upper-bound constraint? Our CI will test it for you.

Reply:
I haven't tested with OCaml 4.11; I'm just being conservative when it hasn't been tested.
And I haven't tested with 4.11 because I thought it wasn't out yet. Is it? Opam only shows me alpha/beta/trunk versions among the available switches. Should I use one of those? Which?

Reply:
The latest beta is fine, but we have it running in our CI already. I loosen up the constraint, we'll see what it says.